src/test/java/core/basesyntax/utils/ListUtilTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.utils;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.impl.FruitStorageDao;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.strategy.impl.TransactionStrategyImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ListUtilTest {
+    private ListUtil listUtil;
+    private Map<String, Integer> storageMap;
+    private StorageDao storageDao;
+
+    @Before
+    public void setUp() {
+        FruitStorage fruitStorage = new FruitStorage();
+        storageDao = new FruitStorageDao(fruitStorage);
+        storageMap = fruitStorage.getStorage();
+        listUtil = new ListUtil();
+    }
+
+    @Test
+    public void processList() {
```

Comment:
You should name all your tests in the same way. If you already have other tests named like methodUnderTest_state_expectedBehavior, then this one should be named like this too